package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "codex.docs",
   "license": "Apache-2.0",
-  "version": "v2.0.0-rc.4",
+  "version": "v2.2.0-rc.1",
```

Comment:
why not 2.1.0-rc.1?

Reply:
because it is release candidate for the next version (current is 2.1.0)

Reply:
package.json says that current is 2.0.0, is it wrong?

Reply:
yes it is
it was not published and usage docs as npm package is not documented yet (this is beta feature)